modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/jdbc/JdbcConnectionContext.java

```diff
@@ -66,8 +66,11 @@
     /** Version 2.8.1: adds query memory quotas.*/
     static final ClientListenerProtocolVersion VER_2_8_1 = ClientListenerProtocolVersion.create(2, 8, 1);
 
+    /** Version 2.9.0: adds custom object support. */
+    static final ClientListenerProtocolVersion VER_2_9_0 = ClientListenerProtocolVersion.create(2, 9, 0);
```

Comment:
Why not `2.8.2`?

Reply:
because I misunderstood the strategy of versioning. Change to 2.8.2